tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "es2020",
+    "lib": ["dom", "es2020"],
+    "target": "es2020",
```

Comment:
❓ during discussions in https://github.com/maxGraph/maxGraph/pull/123#discussion_r990617667 you suggested to switch from es2020 to es6.
Is this something that you want to do in a dedicated PR or did you change your mind?

Reply:
I think it's difficult decision to make. Most modern browsers will work with es2020 just fine, so we can stick with `es2020` for now until supported browsers range is defined.

Reply:
OK, that's make sense.